versionpb/hack.go

```diff
@@ -0,0 +1,7 @@
+package main
```

Comment:
I `main` package name can cause problems, and you don't really have to use that name and `func main`.